docs/standard/microservices-architecture/implement-resilient-applications/implement-custom-http-call-retries-exponential-backoff.md

```diff
@@ -1,17 +1,17 @@
 ---
-title: Implementing custom HTTP call retries with exponential backoff
-description: .NET Microservices Architecture for Containerized .NET Applications | Implementing custom HTTP call retries with exponential backoff
+title: Explore custom HTTP call retries with exponential backoff
```

Comment:
do we need to rename this?

Reply:
This section is meant to be named "Explore" because it is not how you should implement it as we have better API. This is an initial exploration if you had to do it from scratch, so you can compare.
I'm renaming the files to explore, ok? :)